airflow/operator/rbac/leader_election_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
We should check what this is for since it is trying to put something in namespace `system`

Reply:
There are 2 files related to this `leader_election_role_binding.yaml` and `leader_election_role.yaml`, I could not understand what the purpose is.

Reply:
Ok, so this is used if you need to scale the operator - the name of the one that is elected as leader is put into a CM, so that is why it needs to be able to work with ConfigMaps...So, this is fine